css-positioning/relative.css

```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  top: 10px;
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 40px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 90px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 160px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 250px;
+}
```

Comment:
You've added `position: relative` to each of your individual CSS rulesets. Do we already have a ruleset that would affect all of these elements? (basically being able to write `position: relative` one time rather than 5 times and it would work if we had more elements added in)